app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragment.kt

```diff
@@ -53,12 +53,16 @@ class DownloadFragment : LibraryPageFragment<DownloadItem>(), UserInteractionHan
     private var pendingDownloadDeletionJob: (suspend () -> Unit)? = null
     private lateinit var downloadsUseCases: DownloadsUseCases
 
+    private var _binding: FragmentDownloadsBinding? = null
+    private val binding get() = _binding!!
+
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
-        val view = inflater.inflate(R.layout.fragment_downloads, container, false)
+    ): View {
+        _binding = FragmentDownloadsBinding.inflate(inflater, container, false)
+        val view = binding.root
```

Comment:
nit: I see this property is only used in `return`. We could just return `binding.root`.